mix.exs

```diff
@@ -6,70 +6,93 @@ defmodule WorkOS.MixProject do
 
   def project do
     [
-      name: "WorkOS SDK for Elixir",
       app: :workos,
       version: @version,
+      name: "WorkOS SDK for Elixir",
       elixir: "~> 1.11",
+      elixirc_paths: elixirc_paths(Mix.env()),
       start_permanent: Mix.env() == :prod,
       description: description(),
       package: package(),
-      deps: deps(),
-      source_url: "https://github.com/workos-inc/workos-elixir/",
-      homepage_url: "https://workos.com",
       docs: docs(),
-      source_ref: "#{@version}",
-      source_url: @source_url
+      deps: deps(),
+      source_ref: "#{@version}"
+    ]
+  end
+
+  defp description do
+    """
+    Official Elixir SDK for interacting with the WorkOS API.
+    """
+  end
+
+  defp package() do
+    [
+      description: description(),
+      licenses: ["MIT"],
+      maintainers: [
+        "Mark Tran",
+        "Laura Beatris",
+        "Blair Lunceford",
+        "Jacobia Johnson"
+      ],
+      links: %{
+        "GitHub" => @source_url,
+        "WorkOS" => "https://workos.com",
+        "Elixir Example" => "https://github.com/workos/elixir-example-applications"
+      }
     ]
   end
 
+  # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:logger],
-      env: env()
+      extra_applications: [:logger]
     ]
   end
 
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
+  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:tesla, "~> 1.4"},
       {:hackney, "~> 1.18.0"},
       {:jason, ">= 1.0.0"},
       {:plug_crypto, "~> 1.0"},
       {:ex_doc, "~> 0.23", only: :dev, runtime: false},
-      {:credo, "~> 1.6", only: [:dev, :test], runtime: false}
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 1.2", only: :dev, runtime: false},
```

Comment:
The [Dialyzer](http://erlang.org/doc/man/dialyzer.html), a "DIscrepancy AnaLYZer for ERlang programs" is a tool for static code analysis. 

In other words it reads but do not run code and analyse it e.g. looking for some bugs, dead, unnecessary or unreachable code.

The [Dialyxir](https://github.com/jeremyjh/dialyxir) is a mix task to simplify usage of Dialyzer in Elixir.